### DISCUSSION TITLE
Awesome OpenSCAD

### DISCUSSION THREAD

**Discussion by elasticdotventures** [jscad/OpenJSCAD.org#1080] [Ideas]
Hi, I'm [https://github.com/elasticdotventures/openscad-awesome]()

I've got jscad/OpenJSCAD.org on the list, no worries there.

I'm in the process of curating a lot of the OpenSCAD community, and I would like to know the level of interest in either incorporation or collaborating on a new cross-platform object & tool browser/discovery platform & package manager installer built in RUST/wasm.

This way OpenSCAD objects can move between engines a bit easier and I plan to use standard ci/cd build pipelines to trigger git actions (provide examples) to update other software as versions change or are improved each community receives the updates with significantly less toil.   

The idea is to provide a fundamental set of building blocks and let each distinct engine with openscad embedded may have it's own branch hierarchy within the common repo.   Specifically I would propose that the awesome repo will pull & perform integration tests against changes from a branch inside jscad (i.e. design the JSCad & awesome OpenSCAD library to work harmoniously)

*Long term I would also like to form relationships with other object libraries such as thingiverse, kicad, blender etc.  to allow import/export.  I'm hoping by reducing the curation responsibility of scad libraries & tools to a single installable awesome repo.

**Comment by z3dev** [comment]
@elasticdotventures JSCAD has broken away from OpenSCAD for many reasons. Mostly because of limitations in the underlying language and libraries.

We already know of both C++ and C# ports of JSCAD, so RUST should be possible.
